orchestra/static/orchestra/common/js/orchestra_api.js

```diff
@@ -0,0 +1,62 @@
+var serviceModule =  angular.module('orchestra.common.services');
+
+serviceModule.factory('orchestraApi', ['$http', function($http) {
+  var actionService = new function() {
+    var actionService = this;
```

Comment:
Not right now—was planning on adding in the rest in a separate PR, let me know if it should go in this one.